__tests__/components/Button.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import { Button } from '../../src/components';
+
+describe('Button', () => {
+  const onPress = () => {
+    return;
+  };
+
+  it('renders a landscape button', () => {
```

Comment:
why do you think that invert has something to do with landscape? its wrong. please check what invert does. i think the snapshots could tell. if not, try out two buttons (one invert one not) in the app.

Reply:
yes right, Outline is one of the three default styles of RNEButton. Correction made in 61c542c168e0cf5d5715f440ab76cfabfb8ccf7e